lib/core/src/lib/info-drawer/info-drawer-layout.component.scss

```diff
@@ -14,7 +14,7 @@ $adf-info-drawer-icon-size-half: 24px !default;
 
         overflow: auto;
         width: 100%;
-        background-color: var(--theme-background-color);
+        background-color:var(--theme-card-background-color);
```

Comment:
Add a space. In the styles, the format should be 'property:<sapce>value'.

Reply:
Done